### PR TITLE
Modify list automation API

### PR DESCRIPTION
**Pull request by varsnaga**
@vendasta/external-apis

- Modify List API specifications
**TODO**:
- [ ] Review the [pre release checklist](https://vendasta.jira.com/wiki/spaces/API/pages/1651769533/Pre+Release+Checks)

### CODE REVIEW

File: openapi/platform/platform.yaml
Comment (malamsk) [inline openapi/platform/platform.yaml:1]: Because we are not using partner id and market anymore, we should remove these two from docs.

**Review by malamsk — COMMENTED** [review]
Do update the PR title and descriptions for more context.

**Review by malamsk — APPROVED** [review]

File: openapi/platform/platform.yaml [hunk 1]
```diff
@@ -2062,13 +2056,10 @@ paths:
...
-            type: array
-            items:
-              type: string
-            example: default
+            type: string
           in: query
-          name: 'filter[marketIds]'
-          description: Returns automations matching the provided market IDs.
+          name: 'filter[namespace]'
+          description: Returns automations for the provided namespace
```
Comment (richard-rance) [inline openapi/platform/platform.yaml:2062]: Partners do not know what a namespace is. We should clarify that it is an account id or partner id.

**Review by malamsk — APPROVED** [review]

**Review by charanVendasta — APPROVED** [review]
LGTM

**Review by aditya-ven — APPROVED** [review]

**Review by nileshsumbvendasta — COMMENTED** [review]
Looks good

**Pull request merged**